demos-europe/demosplan-core
# chore: autowire repositories via compiler pass

Improve the developer experience by using a compilerPass to autowire all repository entity classes automatically even if they extend the `FluentRepository` Only special cases where the entity is not named as the repository need to be defined in `services_autowire.yml`

### How to review/test
code review or checkout and run. It should explode immediately if somethings goes wrong





### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
